src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,25 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { rivetize, shortuid } from '../Rivet';
+
+interface PanelProps {
+
+    /**
+     * If true, adds a class to create a panel with a white background
+     * See https://rivet.uits.iu.edu/components/layout/panels/#light-modifier
+     */
+    light?: boolean;
+}
+
+const Panel : React.SFC<PanelProps & React.HTMLAttributes<HTMLDivElement>> = ({ children, className, id=shortuid(), light, ...attrs }) => {
```

Comment:
Does this need to generate an ID?  The only components that generate an automated ID are those that need it for things like ARIA labels

Reply:
Ah, no. I'll remove it.